Majasdarbs/10-Galadarbs/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "checkifsimilar",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build-css": "tailwind build -i src/styles.css -o public/styles.css --watch",
+    "watch": "watch \"npm run build-css\""
```

Comment:
you don't need this command, I think build-css already watches for tailwind changes.
The only thing - you need to change the command to this 
npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch